### ISSUE TITLE
something is wrong with xspress3 data collection.

### ISSUE THREAD

**Issue by MehmetTopsakal** [NSLS-II-XPD-tomo/profile_collection#4]
Hi @mrakitin @jklynch @tacaswell 

I think something is wrong with [40-xspress3.py](https://github.com/NSLS-II-XPD-tomo/profile_collection/blob/xspress3-community-ioc/startup/40-xspress3.py)

I want to demonstrate it with a real data.

I collected ```1c6393d9``` like this:

```python
RE(count([xs3], num=200))[0]
```

which is a count using xspress3.

```python
from tiled.client import from_profile
raw = from_profile('xpdd')
import matplotlib.pyplot as plt


uid =  '1c6393d9' #
ds = raw[uid]['primary'].read()
ds['time'] = ds['time'] - ds['time'][0]


fig = plt.figure(figsize=(16,6),dpi=96)


ax_1l = fig.add_subplot(1,3,1)  
ax_1r = ax_1l.twinx()           

ax_1l.yaxis.label.set_color('r')
ax_1r.yaxis.label.set_color('b')

ax_1l.tick_params(axis='y', colors='r') 
ax_1r.tick_params(axis='y', colors='b') 

ds.xs3_channel01_mcaroi01_total_rbv.plot(ax=ax_1l,marker='s',color='r')
ds.xs3_channel01_data.sum(axis=1).plot(ax=ax_1r,marker='o',color='b')
ax_1r.set_ylabel('manual sum')

ax_2 = fig.add_subplot(1,3,2)  #
ds.xs3_channel01_data.plot.imshow(ax=ax_2,robust=True)



ax_3u = fig.add_subplot(2,3,3)  #
for i in ds.xs3_channel01_data:
    i.plot()
ax_3u.set_ylim(bottom=-10,top=10000)
ax_3u.set_title(None)


ax_3l = fig.add_subplot(2,3,6)  #
ds.xs3_channel01_data.sel(time=0,method='nearest').plot(label='time=0')
ds.xs3_channel01_data.sel(time=100,method='nearest').plot(label='time=100')
ds.xs3_channel01_data.sel(time=150,method='nearest').plot(label='time=150')
ds.xs3_channel01_data.sel(time=250,method='nearest').plot(label='time=250')
ds.xs3_channel01_data.sel(time=300,method='nearest').plot(label='time=300')
ax_3l.legend()
ax_3l.set_ylim(bottom=-10,top=10000)


plt.tight_layout()
```

produces plot below

![image](https://user-images.githubusercontent.com/12613640/205153461-1cf8eaf5-34cd-4694-8ce7-56f397096ce6.png)

The red line in subplot 131 makes sense. Because NSLS2 went into maintaince mode at ~time=240. 
Problem is the blue line. It should be constant as red line until beam is off. Then it should drop like the red line.
Since there is no beam, xspress3 should report ~zero counts after bem is off. 

Basically, time=300 spectrum on bottom right subplot should be similar to time=0 spectrum. As in the plots, there is an accumulation. 

This is not only specific to this run. All of my ```RE(count...``` ```RE(scan...``` have the same problem.

Any thoughts?

Thanks
Mehmet

**Comment by mrakitin** [comment]
See the TODOs at https://github.com/NSLS-II-XPD-tomo/profile_collection/pull/3#issuecomment-1347210528.

**Comment by mrakitin** [comment]
xref https://jira.nsls2.bnl.gov/browse/HXSS-432

**Comment by mrakitin** [comment]
Addressed via #3 and the 2d3c26caa10c58dde67565ce71d2072f4a8b261a & a7d97c27fc0e036bf25fc5252ee3c12d95203d77 commits.
